NitroxServer-Subnautica/Communication/Packets/Processors/RocketStageUpdateProcessor.cs

```diff
@@ -25,7 +25,15 @@ public override void Process(RocketStageUpdate packet, NitroxServer.Player playe
 
             if (opRocket.HasValue)
             {
-                opRocket.Value.CurrentStage = packet.NewStage;
+                if (packet.NewStage > opRocket.Value.CurrentStage)
+                {
+                    opRocket.Value.CurrentStage = packet.NewStage;
+                }
+                else
+                {
+                    Log.Error($"{nameof(RocketStageUpdateProcessor)}: Received invalid data to update existing rocket {packet.Id}");
```

Comment:
Error should be descriptive i.e. it should mention the stages not appearing in the right order (I assume).